config/services_test.yaml

```diff
@@ -0,0 +1,39 @@
+parameters:
+  mlp.packages.dir: '%kernel.project_dir%/tests/Common/packages'
+  mlp.translations.dir: '%kernel.project_dir%/tests/Common/translations'
+  mlp.transifex.api.token: 'some_api_token'
+  mlp.transifex.organisation: 'some_organisation'
+  mlp.transifex.project: 'some_project'
+  mlp.aws.key: 'some_aws_key'
+  mlp.aws.secret: 'some_aws_secret'
+  mlp.aws.region: 'some_aws_region'
+  mlp.aws.bucket: 'some_aws_bucket'
+
+services:
+  transifex.test.client:
+    class: MauticLanguagePacker\Tests\Common\Client\TransifexTestClient
+    public: true
```

Comment:
Can we use Guzzle's [MockHandler](https://docs.guzzlephp.org/en/stable/testing.html#mock-handler)?